scripts/k8s-deployment-sim.sh

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+printf "\n\n\n*** [0] install gateway with HA in k8s\n"
+yq e '.global.agtK8Config.withPlugins.tls.enabled = false' -i k8s/agent+lber/values.yaml
+yq e '.global.agtK8Config.withPlugins.vln.enabled = false' -i k8s/agent+lber/values.yaml
+helm install my-app k8s/agent+lber --set-file global.agtConfig=k8s/example/gateway.env
+printf "\n\n\n*** [0.1] verify installation\n"
+kubectl get deployments && kubectl get sts && kubectl get pods && kubectl get services && kubectl get ingress
+printf "\n\n\n*** [0.2] verify deployment spec\n"
```

Comment:
add ingress spec checking to benefit future debugging.

Reply:
Update the script